modules/ListView.js

```diff
@@ -156,10 +157,10 @@
         injectionSite = this.$el;
       args = args || {};
       this._modelName = args.childModel || 'model';
-      this._collection = args.collection;
-      this._childView = args.childView;
-      this._template = args.template;
-      this._childrenContainer = args.childrenContainer;
+      this.collection = args.collection || this.collection;
+      this._childView = args.childView || this.childView;
+      this._template = args.template || this.template;
+      this._childrenContainer = args.childrenContainer || this.childrenContainer;
```

Comment:
These defaults should have the same visibility in both places.  For backwards compatibility I'd suggest:
  this._childView = args.childView || this._childView

These are different because there isn't already a backbone concept for them (like collection).